test/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ addtest(cleanup)
 addtest(color_diagnostics)
 addtest(config)
 addtest(cpp1)
+addtest(compile_command)
```

Comment:
The compile_commands suite fails in all CI jobs that execute it. Please have a look.